flink-runtime/src/test/java/org/apache/flink/runtime/state/memory/ByteStreamStateHandleTest.java

```diff
@@ -172,4 +172,15 @@ public void testBulkReadINdexOutOfBounds() throws IOException {
             // expected
         }
     }
+
+    @Test
+    public void testStreamWithEmptyByteArray() throws IOException {
+        final byte[] data = new byte[0];
+        final ByteStreamStateHandle handle = new ByteStreamStateHandle("name", data);
+
+        FSDataInputStream in = handle.openInputStream();
+        in.seek(0);
```

Comment:
Why we need to seek to the 1st position?

Reply:
this can be removed.

Reply:
> Why we need to seek to the 1st position?

I will remove it.